source/Planet.cpp

```diff
@@ -408,6 +409,48 @@ const Sale<Outfit> &Planet::Outfitter() const
 
 
 
+// Get the local price of this outfit.
+double Planet::GetLocalRelativePrice(const Outfit &outfit, const ConditionsStore &conditions) const
+{
+	int days = conditions.Get("year") + conditions.Get("month") + conditions.Get("day");
```

Comment:
This is... interesting. What are you trying to do here?

Reply:
Only refresh it once per day